guides/blending/client-side-blending/README.md

```diff
@@ -0,0 +1,52 @@
+# Client-Side Blending
+
+When merging responses from different apps with [server-side blending](server-side-blending), the views are stacked on top of each other. In most cases, that's not what we want. Instead, we would like to compose elements in the views to make the result look like one page. That's what client-side blending does:
+
+![Blending comparison](/assets/BlendingComparison2.png)
+
+## Composing Views
+ 
+To solve this, we could edit the source code of the apps involved. This is impractical, and in some cases even impossible. The main reason is that the apps in a system, or application suite, are often created by different developers and it's not always possible to get access to their source code. A secondary reason is versatility. If an app is modified to work well with your application suite, it might not work well with other application suites.
+
+Client-side blending lets you change the structure of blended apps without touching the source code. It does this by swapping the default composition of HTML that's sent to the client with an alternative composition. This requires a couple of things:
+1. Alternative compositions should be defined with the default composition as a starting point. Otherwise, the HTML would have to be rewritten which would be inefficient, especially since alternative compositions are often similar to default compositions.
+2. Alternative compositions have to be tied to a specific set of responses. For example, if you have four apps in your app suite and you swap out one of them, you wouldn't want the composition for the previous set of apps to still apply.
+3. Alternative compositions have to be stored persistently. If an app is restarted, the defined compositions should still be applied.
+4. Alternative compositions have to be applied when the different responses are merged and not on application start since server-side blending can change in runtime.
+
+### 1. Defining Alternative Compositions
+
+Alternative compositions are defined in the browser when the apps are running with the [CompositionEditor](https://github.com/StarcounterApps/CompositionProvider). It looks like this:
```

Comment:
Ah! now I get it! 
But I'm afraid only because I've made it ;) so I know what's there.


so "Alternative composition" is just "Shadow DOM layout composition, a piece of HTML that will be used to replace initial/default one, and preferably distribute other content/functional elements."?